typescript/graphql-hapi-sdl-first/package.json

```
@@ -6,10 +6,11 @@
     "dev": "ts-node src/index.ts"
   },
   "dependencies": {
+    "@apollo/server": "^4.7.1",
+    "@as-integrations/hapi": "^2.0.0",
```

I think we should pin these? (after discussion in Slack etc etc)
```suggestion
    "@apollo/server": "4.7.1",
    "@as-integrations/hapi": "2.0.0",
```